plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -773,6 +773,17 @@ public Optional<Object> getInfo(ConnectorTableHandle tableHandle)
     @Override
     public List<SchemaTableName> listTables(ConnectorSession session, Optional<String> optionalSchemaName)
     {
+        if (optionalSchemaName.isEmpty()) {
+            Optional<List<SchemaTableName>> allTables = metastore.getAllTables();
+            if (allTables.isPresent()) {
+                return ImmutableList.<SchemaTableName>builder()
+                        .addAll(allTables.orElseThrow().stream()
```

Comment:
orElseThrow -> get

this provides IDE support (warning if `isPresent` is later removed)

Reply:
Done